application/features/bootstrap/EmailContext.php

```diff
@@ -1059,6 +1061,14 @@ public function aManagerRescueEmailIsSentToTheManager()
         $this->assertEmailSent(EmailLog::MESSAGE_TYPE_MFA_MANAGER, self::MANAGER_EMAIL);
     }
 
+    /**
+     * @Then a Recovery Rescue email is sent to the recovery contact
+     */
+    public function aRecoveryRescueEmailIsSentToTheRecoveryContact()
+    {
+        $this->assertEmailSent(EmailLog::MESSAGE_TYPE_MFA_RECOVERY, $this->testMfaOption->recovery_email);
```

Comment:
It seems like you could simplify this to just use the constant (like was done with the manager rescue email):
```suggestion
        $this->assertEmailSent(EmailLog::MESSAGE_TYPE_MFA_RECOVERY, self::RECOVERY_EMAIL);
```
Was there a benefit to using the `testMfaOption` test property instead?

Reply:
testMfaOption was only there because we had it stored there, since we're removing that from being stored, it would be better to do what you're suggesting.